src/domain/shared/components/Comments/commentsDetails.graphql

```diff
@@ -0,0 +1,10 @@
+fragment CommentsDetails on Comments {
```

Comment:
Rename to `CommentsWithMessages` or something related to be more explicit about the contents. `Details` is vague and prooved inefficient for understanding the codebase.